src/planscape/plan/tests.py

```diff
@@ -298,6 +298,24 @@ def test_list_plans_by_owner_no_user(self):
             else:
                 self.assertTrue(False)
 
+    def test_list_plans_by_owner_no_user_logged_in(self):
+        self.client.force_login(self.user)
+        response = self.client.get(reverse('plan:list_plans_by_owner'), {},
+                                   content_type="application/json")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.json()), 2)
+        for plan in response.json():
+            self.assertTrue('geometry' not in plan)
+            self.assertEqual(plan['region_name'], 'Sierra Nevada')
+            if plan['name'] == 'plan3':
```

Comment:
just thoughts on test readability:
1) there's no partiallyEquals operator like there is for protobufs - "assertDictEqual(b, b | a)" might help.
2) create_plan could be called within each test rather than in the set up.

but I also see the test style was inherited and changing all tests in this file doesn't seem worth the effort.

Reply:
Ok.  I'll give myself a TODO to fix this, and hopefully get to it!